product_unique_serial_picking/model/stock.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+
+from openerp import _, api, fields, exceptions, models
```

Comment:
This file is duplicated with:
https://github.com/Vauxoo/stock-logistics-workflow/blob/8.0/product_unique_serial/model/stock.py